pulumi/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pulumi>=3.0.0,<4.0.0
```

Comment:
Please pin specific package versions. It is required for a reproducible build. Same goes to the next line.